src/applications/login/components/VerifyPage.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import recordEvent from 'platform/monitoring/record-event';
+
+export default function VerifyPage() {
+  return (
+    <div className="row">
+      <AlertBox
+        content={
+          <div>
+            <h4 className="usa-alert-heading">
+              Please verify your identity before continuing to My VA Health
+            </h4>
+            <p>This process should take about 5 to 10 minutes.</p>
+            <p>
+              <strong>If you’re applying for the first time</strong>
+            </p>
+            <p>
+              We need to verify your identity so we can help you track the
+              status of your application once you’ve submitted it. As soon as
```

Comment:
I don't think this copy makes sense in the context of My VA Health. especially the "applying for the first time/applied before" stuff, the user is not applying for anything at this point. 

Probably a message more like src/platform/site-wide/cta-widget/components/messages/Verify.jsx would make more sense.